backend/src/db/seed.js

```diff
@@ -693,25 +699,80 @@ const languages = ['de', 'en', 'es', 'fr', 'it', 'pt', 'pl']
         reasonDescription: 'This user is harassing me with bigoted remarks!',
       }),
       reportAgainstDagobert.relateTo(dagobert, 'belongsTo'),
-      reportAgainstTrollingPost.relateTo(jennyRostock, 'filed', {
+
+      reportAgainstHuey.relateTo(jennyRostock, 'filed', {
+        resourceId: 'u4',
+        reasonCategory: 'doxing',
+        reasonDescription: '',
+      }),
+      reportAgainstHuey.relateTo(huey, 'belongsTo'),
+
+      reportAgainstTrollingPostP2.relateTo(jennyRostock, 'filed', {
         resourceId: 'p2',
         reasonCategory: 'doxing',
         reasonDescription: "This shouldn't be shown to anybody else! It's my private thing!",
       }),
-      reportAgainstTrollingPost.relateTo(p2, 'belongsTo'),
-      reportAgainstTrollingComment.relateTo(huey, 'filed', {
+      reportAgainstTrollingPostP2.relateTo(p2, 'belongsTo'),
+
+      reportAgainstTrollingPostP9.relateTo(jennyRostock, 'filed', {
+        resourceId: 'p9',
+        reasonCategory: 'discrimination_etc',
+        reasonDescription: 'Discrimination !!!',
+      }),
+      reportAgainstTrollingPostP9.relateTo(p9, 'belongsTo'),
+
+      reportAgainstTrollingCommentC1.relateTo(huey, 'filed', {
         resourceId: 'c1',
         reasonCategory: 'other',
         reasonDescription: 'This comment is bigoted',
       }),
-      reportAgainstTrollingComment.relateTo(trollingComment, 'belongsTo'),
+      reportAgainstTrollingCommentC1.relateTo(trollingCommentC1, 'belongsTo'),
+
+      reportAgainstTrollingCommentC2.relateTo(jennyRostock, 'filed', {
+        resourceId: 'c2',
+        reasonCategory: 'other',
+        reasonDescription: 'This comment is bigoted',
+      }),
+      reportAgainstTrollingCommentC2.relateTo(trollingCommentC2, 'belongsTo'),
+
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2020-02-01T14:13:50Z" title="Saturday, February 1st 2020, 3:13:50 pm +01:00">Feb 1, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -693,25 +701,80 @@ const languages = ['de', 'en', 'es', 'fr', 'it', 'pt', 'pl']
         reasonDescription: 'This user is harassing me with bigoted remarks!',
       }),
       reportAgainstDagobert.relateTo(dagobert, 'belongsTo'),
-      reportAgainstTrollingPost.relateTo(jennyRostock, 'filed', {
+
+      reportAgainstHuey.relateTo(jennyRostock, 'filed', {
+        resourceId: 'u4',
+        reasonCategory: 'doxing',
+        reasonDescription: '',
+      }),
+      reportAgainstHuey.relateTo(huey, 'belongsTo'),
+
+      reportAgainstTrollingPostP2.relateTo(jennyRostock, 'filed', {
         resourceId: 'p2',
         reasonCategory: 'doxing',
         reasonDescription: "This shouldn't be shown to anybody else! It's my private thing!",
       }),
-      reportAgainstTrollingPost.relateTo(p2, 'belongsTo'),
-      reportAgainstTrollingComment.relateTo(huey, 'filed', {
+      reportAgainstTrollingPostP2.relateTo(p2, 'belongsTo'),
+
+      reportAgainstTrollingPostP9.relateTo(jennyRostock, 'filed', {
+        resourceId: 'p9',
+        reasonCategory: 'discrimination_etc',
+        reasonDescription: 'Discrimination !!!',
+      }),
+      reportAgainstTrollingPostP9.relateTo(p9, 'belongsTo'),
+
+      reportAgainstTrollingCommentC1.relateTo(huey, 'filed', {
         resourceId: 'c1',
         reasonCategory: 'other',
         reasonDescription: 'This comment is bigoted',
       }),
-      reportAgainstTrollingComment.relateTo(trollingComment, 'belongsTo'),
+      reportAgainstTrollingCommentC1.relateTo(trollingCommentC1, 'belongsTo'),
+
+      reportAgainstTrollingCommentC2.relateTo(jennyRostock, 'filed', {
+        resourceId: 'c2',
+        reasonCategory: 'other',
+        reasonDescription: 'This comment is bigoted',
+      }),
+      reportAgainstTrollingCommentC2.relateTo(trollingCommentC2, 'belongsTo'),
+
```


With #2921 it will be much easier to implement a `report` factory that sets up all the required data and relationships.

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-02-05T13:14:51Z" title="Wednesday, February 5th 2020, 2:14:51 pm +01:00">Feb 5, 2020</time>_

---


I am curious how that work then. 🤗
Just had a look to the first files of this PR till now.